docs/_data/menu-ml.yaml

```diff
@@ -1,10 +1,10 @@
-- text: Feature extraction, transformation, and selection
+- text: "Overview: estimators, transformers and pipelines"
+  url: ml-intro.html
+- text: Building and transforming features
```

Comment:
I like using the keywords "extraction, transformation, and selection" since users may search for those.  "Building" is pretty generic.